src/Platforms/AbstractMySQLPlatform.php

```diff
@@ -642,8 +642,15 @@ public function getAlterTableSQL(TableDiff $diff)
             $columnArray = $column->toArray();
 
             $columnArray['comment'] = $this->getColumnComment($column);
-            $queryParts[]           =  'CHANGE ' . ($columnDiff->getOldColumnName()->getQuotedName($this)) . ' '
-                    . $this->getColumnDeclarationSQL($column->getQuotedName($this), $columnArray);
+
+            if ($columnDiff->fromColumn !== null) {
+                $oldColumn = $columnDiff->fromColumn;
```

Comment:
I think it would be best to use `oldColumn` and `oldColumnName` as little as possible, and switch to `fromColumn` / `fromColumnName` as soon as possible (to minimize the diff between branches)

Reply:
I wouldn't say that the from/to naming is better than old/new. What diff between the branches are you referring to?

Reply:
> I wouldn't say that the from/to naming is better than old/new (this is what GNU diff uses, for example).

I'm not saying it's better, but since we are using from/to as well, it might make sense to use the same naming everywhere.

> What diff between the branches are you referring to?

The diff you would obtain when doing that kind of cleanup after merging up.

Reply:
So if I understand it correctly, it's about the variable names being used in the code, not any sort of API? Please see the updated version of the patch.